server/server.js

```diff
@@ -14,34 +14,30 @@ app.get('/similar/:product_id', async (req, res) => {
   const { product_id } = req.params;
   try {
     const associatedProductNumbers = await db.getAssociatedProductNums(product_id)
+    const allInfo = await Promise.all(associatedProductNumbers.map(async (productNum) => {
```

Comment:
@masoncpott I need to look up this async and await thing. You are killing it here!